isb_web/config.py

```diff
@@ -48,5 +48,11 @@ class Settings(BaseSettings):
         r"https?://localhost(:\d{2,5})/",
     ]
 
+    thing_url_path: str = "thing"
+
+    stac_item_url_path: str = "stac_item"
+
+    stac_collection_url_path: str = "stac_collection"
+
```

Comment:
Since we were needing to generate links that match the url paths in the FastAPI app, consolidate these into settings.